docs/excel/custom-functions-troubleshooting.md

```diff
@@ -72,6 +72,10 @@ CustomFunctions.associate("ADD", add);
 
 For more information on this process, see [Associating function names with JSON metadata](../excel/custom-functions-json.md#associating-function-names-with-json-metadata).
 
+## Known issues
```

Comment:
There's something slightly off about having  "Common issues" precede "Known issues". The latter is using "issues" to mean bugs or deficiencies, whereas the former is referring to common design pitfalls.

Reply:
Do you have alternate phrasing suggestions for either location? "Known issues" is oblique but it's how we title the bugs sections in add-ins docs and is in line with the industry standard, so I'm inclined to change "Common issues" instead. What do you think of "Common problems"? It avoids the confusing usage of "issues" but unfortunately is similarly vague.

Reply:
Agreed that "Common issues" is the part to change. I like "Common problems", though perhaps "Common problems and solutions" sounds more enticing.

Reply:
Great idea!